### ISSUE TITLE
Remove dnode dependency

### ISSUE THREAD

**Issue by Apollon77** [TuyaAPI/cli#4]
... you can use it like:

Rule:
```
module.exports = function(callback) {
    return {
        summary: 'Catch Tuya Data from Response',
        *beforeSendResponse(requestDetail, responseDetail) {
            const body = responseDetail.response.body.toString('utf8');
            if (body.includes('tuya.m.my.group.device.list')) {
                try {
                    const response = JSON.parse(body);
                }
                catch (err) {
                    return null;
                }
                callback(response);
                return null;
            }
        },

        *beforeDealHttpsRequest(requestDetail) {
            return requestDetail.host.includes('tuya');
        }
    };
};
```

And in main file:
```
rule: require('./lib/anyproxy-rule.js')(resonse => { ... }),
```

**Comment by codetheweb** [comment]
Wow, that's a lot smarter than what I was doing.  Thanks.

I'll change this later today.

**Comment by Apollon77** [comment]
Yes, found that way after experimenting a bit while implementing in my ioBroker adapter directly.

In fact i defined a function and then have the function name as parameter instead of the function definition because it feeled weired in this "object definition context" :-)

**Comment by codetheweb** [comment]
Fixed in v1.4.0.
